src/main/java/com/alienfast/bamboozled/ruby/config/RubyBuildConfigurationPlugin.java

```diff
@@ -85,6 +86,11 @@ public boolean isApplicableTo( Plan plan ) {
     }
 
     @Override
+    public boolean isApplicableTo(@NotNull ImmutablePlan plan) {
+        return false;
```

Comment:
I also missed to answer: The dropdown is shown on the Misc tab.

![screenshot](https://frupic.frubar.net/shots/34656.png)